bundles/model/org.eclipse.smarthome.model.lsp/src/main/java/org/eclipse/smarthome/model/lsp/internal/ModelServer.java

```diff
@@ -82,7 +82,7 @@ private void listen() {
                         handleConnection(client);
                     }, "Client " + client.getRemoteSocketAddress()).start();
                 } catch (IOException e) {
-                    logger.error("Error accepting the client connection", e);
+                    logger.debug("Error accepting the client connection: {}", e.getMessage());
```

Comment:
IMHO it depends if a a closed socket is the only reason for an `IOException`.
Do you know if it is the only one?

If it is not the only possible reason, we should check if the exception is an instance of `java.net.SocketException` and if the socket is closed. In this case we can use debug level or not logging anything.

If there are other cases and it is not a closed socket, we should keep level error.

WDYT?

Reply:
Imho all other reasons are potentially due to the client not doing a proper connection/handshake. And this would be similar like we have HTTP clients accessing our HTTP server - we do not log such requests as an ERROR as the bug is on the client side, not on our side. So I would think that debug is the right choice here.

Reply:
And what about
```
java.net.SocketException: Too many open files
```
this is an error on the server side that could identify a resource leak.

And it is rather easy to run out of free file handles using some third party code :wink:

Reply:
Ok, you won 🥇 
But how shall we proceed here then? It seems as if the only option is to parse the exception message, because otherwise I cannot differentiate beween "Too many open files" and "Socket closed".